GVFS/GVFS.Common/Database/GVFSDatabase.cs

```diff
@@ -0,0 +1,135 @@
+﻿using GVFS.Common.FileSystem;
+using GVFS.Common.Tracing;
+using Microsoft.Data.Sqlite;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+
+namespace GVFS.Common.Database
+{
+    public class GVFSDatabase : IDisposable

```

Comment:
Consider including a class comment giving a brief description of what a `GVFSDatabase` class represents.